packages/react-component-library/src/intro/design-guide.mdx

```diff
@@ -2,33 +2,170 @@ import { Meta, Source } from '@storybook/blocks'
 
 <Meta title="Designers" />
 
-# Design Principles
+# Axure Design Library
 
-Here’s a checklist of everything you need to get started:
+Designers can use our Axure libraries to quickly prototype, test and validate ideas with users. There are 2 libraries:
 
-## Design tokens
-The Royal Navy Design System provides tokens for colour, typography, spacing and much more. Tokens make it easy to ensure consistency across defence applications.
+### Design Component library
 
-### The case for using tokens
-The number one reason is consistency<sup>[1](#references)</sup>: the palette of `danger` is the same everywhere. End-users of Royal Navy applications highly benefit from consistency across interfaces, where the colour of danger may save lives.
+Contains RNDS Widget with ready to use interactivity.
 
-Another excellent reason to always use tokens, is that if we adjust the colours of `success`, these changes are automatically made wherever it’s in use, in every single live application. Magic.
+- Ideal for creating rich prototypes or quickly putting together static design mockups
+- Interactivity has been programmed for you, depending on the level of realism you need but some effort may be needed.
 
-#### [Colour Tokens](?path=/docs/tokens-colours--docs)
-We use named colours (e.g. Neutral or Action) instead of colour literals (e.g. blue). Colour palettes represent functions, so that we can make interactions and contexts easy to distinguish.
+[Download RNDS Axure Library v4.0](https://www.)
 
+### Icon Library
 
-#### [Spacing](?path=/docs/tokens-spacing--docs)
-Spacing is a key (and often overlooked) aspect of ensuring your application is easy to read and understand. Too little whitespace will cause components to become visually cramped, therefore decreasing readability. The general rule is to start with whitespace that is too large for an element and then decrease it until it is at an appropriate size. It is far better to have too much whitespace than too little.
+A library of approved iconography for clear visual communication.
 
-#### [Typography](?path=/docs/tokens-typography--docs)
-All Royal Navy Design System applications are set in Lato
+[Download RNDS Axure Icon Library](https://www.)
 
 
----
+## Installing Axure libraries
```

Comment:
there is a funky non-visible character here which prevents the heading from being rendered properly